### PR TITLE
Python package structure

### PR DESCRIPTION
**Pull request by LucasLvy**
You now need to fill the relevant fields in the `setup.py` file https://docs.python.org/3/distutils/setupscript.html
```sh
pip install twine
python3 setup.py sdist
twine upload dist/*
```

### CODE REVIEW

**Comment by RobinLinus** [comment]
Thanks a lot for this PR!! Wondering if there is a way to get rid of the `ZeroSync` folder in `src`? We're trying to keep the directory structure as simple as possible.

File: setup.py [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+
+
+setup(
+    name="FILL",
+    version="FILL",
+    license="FILL",
+    author="FILL",
+    author_email="FILL",
+    packages=["ZeroSync"],
+    package_dir={"ZeroSync": "src/ZeroSync"},
```
Comment (RobinLinus) [inline setup.py:11]: Would it suffice to change this line to get rid of the `ZeroSync` folder in `src`?

Comment (LucasLvy) [reply setup.py:11]: I don't know it might introduce some problems but you can try

**Comment by lucidLuckylee** [comment]
Thanks for the PR. I found a way to keep our current structure and still be able to create a package.

Closed with 8886476

**Pull request closed without merging**